### PR TITLE
SIGNALFX_EXPORTER is Zipkin by default

### PR DESCRIPTION
**Pull request by pellared**
## Why

Use proper defaults for Splunk

## What

- Use `Zipkin` as a default value for `SIGNALFX_EXPORTER`
- Opportunistic `USAGE.md` formatting fixes.

## Tests

1. Integration tests
2. Locally `dev/envvar.sh` does not set SIGNALFX_EXPORTER and it works 🎉

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -47,12 +47,12 @@ Use these environment variables to configure the tracing library:
...
-| `SIGNALFX_TRACE_AGENT_URL`, `SIGNALFX_ENDPOINT_URL` | The URL to where trace exporters (see: `SIGNALFX_EXPORTER`) send traces. | `http://localhost:8126` | 
+| `SIGNALFX_TRACE_ENABLED` | Enable to activate the tracer. | `true` |
+| `SIGNALFX_TRACE_DEBUG` | Enable to activate debugging mode for the tracer. | `false` |
+| `SIGNALFX_TRACE_AGENT_URL`, `SIGNALFX_ENDPOINT_URL` | The URL to where trace exporters (see: `SIGNALFX_EXPORTER`) send traces. | `http://localhost:8126` |
 | `SIGNALFX_TAGS` | Comma-separated list of key-value pairs to specify global span tags. For example: `"key1:val1,key2:val2"` |  |
 | `SIGNALFX_LOGS_INJECTION` | Enable to inject trace IDs, span IDs, service name and environment into logs. This requires a compatible logger or manual configuration. | `false` |
-| `SIGNALFX_EXPORTER` | The exporter to be used. The Tracer uses it to encode and dispatch traces. Available values are: `DatadogAgent`, `Zipkin`. | `DatadogAgent` |
+| `SIGNALFX_EXPORTER` | The exporter to be used. The Tracer uses it to encode and dispatch traces. Available values are: `DatadogAgent`, `Zipkin`. | `Zipkin` |
```
Comment (RassK) [inline docs/USAGE.md:55]: 'Available values are: `DatadogAgent`, `Zipkin`.' I would remove this part as first is non important for us and second is default.

Comment (pellared) [reply docs/USAGE.md:55]: @dszmigielski Proposed to add a note that users should not use this configuration. I am considering making a dedicated section (or maybe even markdown?) for internal configuration. What is your preference?

Comment (pellared) [reply docs/USAGE.md:55]: @theletterf any thoughts on how to approach it?

Comment (RassK) [reply docs/USAGE.md:55]: Ok, I thought he meant that we don't remove the entire line. Your idea seems great, minimal successful configuration for public and internally more detailed way, so a new file seems more appropriate. Reduces conflicts for new users.

Comment (pellared) [reply docs/USAGE.md:55]: Please let me do it in a separate PR so that I will try to move more configuration fields there 😉

**Review by RassK — APPROVED** [review]
LGTM

**Review by Kielek — APPROVED** [review]

**Pull request merged**